spark/spark-tensorflow-distributor/Dockerfile

```diff
@@ -52,4 +52,6 @@ RUN apt-get install -y curl && \
 VOLUME /mnt/spark-tensorflow-distributor
 WORKDIR /mnt/spark-tensorflow-distributor
 
+ENV PYTHONPATH=/mnt/spark-tensorflow-distributor
```

Comment:
Without this, test running will fail with error like "do not find module tensorflow_spark_distributor"